From d246f305a196f7e3f4d53328e1194e7c427998f1 Mon Sep 17 00:00:00 2001
From: Zhihao Cheng <chengzhihao1@huawei.com>
Date: Wed, 16 Nov 2022 10:54:00 +0800
Subject: [PATCH 0043/1398] [Huawei] Revert "[Backport] libbpf: Support
 detecting writable tracepoint program"

Offering: HULK
hulk inclusion
category: feature
bugzilla: 187975

--------------------------------

This reverts commit 7204d08d85041599abe493a17e9fc31e3da65691.

Signed-off-by: Zhihao Cheng <chengzhihao1@huawei.com>
---
 tools/lib/bpf/libbpf.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/lib/bpf/libbpf.c b/tools/lib/bpf/libbpf.c
index 2531a2a0ff6c..015ed8253f73 100644
--- a/tools/lib/bpf/libbpf.c
+++ b/tools/lib/bpf/libbpf.c
@@ -8336,7 +8336,6 @@ static const struct bpf_sec_def section_defs[] = {
 		.attach_fn = attach_tp),
 	SEC_DEF("raw_tracepoint/", RAW_TRACEPOINT,
 		.attach_fn = attach_raw_tp),
-	BPF_PROG_SEC("raw_tracepoint.w/", BPF_PROG_TYPE_RAW_TRACEPOINT_WRITABLE),
 	SEC_DEF("raw_tp/", RAW_TRACEPOINT,
 		.attach_fn = attach_raw_tp),
 	SEC_DEF("tp_btf/", TRACING,
-- 
2.17.1

